apps/eo/e2e-eo/src/page-objects/transfers.po.ts

```diff
@@ -26,6 +26,9 @@ export class TransfersPo {
   private newAgreementDateRangeStartInput = '[data-testid="new-agreement-start-date-input"]';
   private newAgreementDateRangeEndInput = '[data-testid="new-agreement-end-date-input"]';
   private newAgreementReceiverInput = '[data-testid="new-agreement-receiver-input"]';
+  private walletDepositEndpointInput = '[data-testid="wallet-deposit-endpoint-input"]';
+  private testBase64EncodedWalletDepositEndpoint =
+    'eyJFbmRwb2ludCI6Imh0dHA6Ly9sb2NhbGhvc3Q6Nzg5MC8iLCJQdWJsaWNLZXkiOiJBVTBWVFVzQUFBQUJ5aE5KRmxENlZhVUZPajRGRzcybmVkSmxVbDRjK0xVejdpV0tRNEkzM1k0Q2J5OVBQTm5SdXRuaWUxT1NVRS9ud0RWTWV3bW14TnFFTkw5a0RZeHdMQWs9IiwiVmVyc2lvbiI6MX0=';
```

Comment:
Minor thing: this is actually a base64-encoded WalletDepositEndpoint. Why not just a random string (maybe a UUID) to signify that it really is just to be thought of as a key/token.

Reply:
Since we are mocking the response for the server, I guess it is possible to just use a random string. I will change it.

Reply:
It's not possible to just insert a random string. There is a validation in place, ensuring the format of the string is a base64 encoded one. I'm keeping the test string as is, for that reason.